packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -98,6 +101,24 @@ const _ActionList = ({ children, surfaceLevel = 2, testID }: ActionListProps): J
   );
   const isMultiSelectable = selectionType === 'multiple';
 
+  const renderActionListItem = React.useCallback(({ item }) => {
+    return <ActionListItem {...item} />;
+  }, []);
+
+  const renderActionListSectionHeader = React.useCallback(({ section: { title } }) => {
+    if (!title) return null;
+    return <ActionListSection title={title} _hideDivider={true} children={undefined} />;
+  }, []);
+
+  const renderActionListSectionDivider = React.useCallback(
+    ({ section: { title, hideDivider } }) => {
+      if (!title) return null;
+      if (hideDivider) return null;
+      return <ActionListSectionDivider />;
```

Comment:
While in the web we wrap multiple ActionListItems inside ActionListSection, in here we can't do that so the 'section header' & 'section footer' components are rendered separately. 

Basically it's a flat structure.